src/server/pkg/deploy/assets/assets.go

```diff
@@ -442,7 +443,35 @@ func PachdService() *v1.Service {
 				{
 					Port:     githook.GitHookPort,
 					Name:     "api-git-port",
-					NodePort: 30000 + githook.GitHookPort,
+					NodePort: githook.NodePort(),
+				},
+			},
+		},
+	}
+}
+
+// GithookService returns a pachd service.
```

Comment:
Ah, nope.

This is the k8s object that is a k8s `svc` that exposes the `650` (grpc port), `651` (go trace port), `30652` (the http get file port), and finally `30999` (the githook port). So it's being added to the right k8s object.

The change made in this PR is to use the `githook.NodePort()` helper since I wanted to make all definitions of the git ports in one place so they don't get out of sync (insted of using the loose convention of just adding 30k to the target port).

Reply:
Ah, I see you're referring to the comment. Yea, that's not a great description. It is not a 'pachyderm service'. It is a k8s object that will allow us to get a public IP/domain for the service on cloud deployments.

I'll update the comment.